frontend/src/components/App/App.jsx

```diff
@@ -50,6 +52,11 @@ const App = () => {
 
     return (
         <Router className="aha__app">
+            { !participant? (
+            <div className="loader-container">
+                <Loading />
+            </div>
+            ) : (
```

Comment:
Is the participant required for all routes or only specific route(s)?

Reply:
In my view, all. There are some Routes here that I don't fully understand though, such as the `URLS.session` one (which is defined no-where).